singelton_classes/os_manager.py

```diff
@@ -0,0 +1,48 @@
+from azure.storage.blob import BlobServiceClient
+from azure.core.exceptions import ResourceExistsError
+from concurrent.futures import ThreadPoolExecutor
+from dotenv import load_dotenv
+import os
+from BionicEye.singelton_classes.singelton_meta import SingletonMeta
+
+
+load_dotenv()
+
+CONNECTION_STRING = os.getenv('CONNECTION_STRING')
+OS_CONTAINER = os.getenv('OS _CONTAINER')
+MAX_THREADS = 100
+
+
+class OSManage(metaclass=SingletonMeta):
+    def __init__(self):
+        """
+        Stores connection to the os and creates the container if not exists
+        """
+        self.blob_service_client = BlobServiceClient.from_connection_string(CONNECTION_STRING)
+        try:
+            self.blob_service_client.create_container('bionic-eye')
+        except ResourceExistsError:
+            pass
+
+    def upload_file(self, file_path):
+        """
+        Uploads a given file to the os
+        :param file_path: the file that will be uploaded
+        """
+        blob_path = os.path.relpath(file_path)
```

Comment:
Why use `relpath` here?

Reply:
the path in the os similar to the relpath